src/components/AppTile.spec.jsx

```diff
@@ -43,11 +43,11 @@ describe('<AppTile />', () => {
   it('renders loading icon when app is in installing state', () => {
     const { getByText } = render(
       <AppLike>
-        <MuiCozyTheme>
+        <CozyTheme>
```

Comment:
We could add the CozyTheme to the AppLike component so as not to repeat it in every test

Reply:
Someone tells me (you), I can't remember where, that we should seperate applike & cozytheme. I was this close to tag you within the commit to tell you "looks how it would have simplified to have cozytheme inside the applike" :D.

Reply:
but maybe it was not about cozytheme, but cozyprovider 🤔